var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -279,6 +281,21 @@ class PyTensorflow(Package, CudaPackage):
     #            type=('build', 'run'), when='@2.8:')
     # depends_on('py-tensorflow-io-gcs-filesystem@0.21:',
     #            type=('build', 'run'), when='@2.7')
+    with when("+rocm"):
```

Comment:
Should we also extend the `ROCmPackage` base class?

Reply:
I have the same question here

Reply:
Sure.. I can add the new versions for rocm as well as extend the ROCmPackage .
If this initial changes are okay, i am planning to bring them in based on the new requests.